launch/gearman-load-logger.yml

```diff
@@ -7,4 +7,5 @@ team: eng-districts
 use_iam_roles: true
 resources:
   cpu: 0.0  # no CPU to improve resource usage (https://clever.atlassian.net/browse/INFRA-2120)
-  max_mem: 0.01
+  soft_mem_limit: 0.01
+  max_mem: 0.02
```

Comment:
yeah. I meant "conservative" as in "less risk of crash / OOM." So bumping it higher 👍